lambda/variables.tf

```diff
@@ -0,0 +1,255 @@
+# --- lambda/variables.tf ---
+variable "prefix" {
+  description = "A prefix used for all resources"
+  type        = string
+}
+
+variable "project" {
+  description = "A name of the project"
+  type        = string
+}
+
+variable "program" {
+  description = "A name of the program"
+  type        = string
+}
+
+variable "env" {
+  description = "The environment to deploy to"
+  type        = string
+}
+
+variable "region" {
+  description = "The region to deploy to"
+  type        = string
+}
+
+variable "lambda_name" {
+  description = "Lambda function name"
+  type        = string
+}
+
+variable "account_id" {
+  description = "Aws account id"
+  type        = string
+}
+
+variable "timeout" {
+  description = "Amount of time your Lambda Function has to run in seconds"
+  default     = 3
+}
+
+variable "source_path" {
+  description = "Lambda function source path"
+  type        = string
+}
+
+variable "lambda_bucket" {
+  description = "S3 bucket on which lambda code reside"
+  type        = string
+}
+
+variable "architectures" {
+  description = "Lambda function architecture"
+  type        = list(string)
+  default     = ["arm64"]
+}
+
+variable "runtime" {
+  description = "Lambda function runtime environment"
+  type        = string
+}
+
+variable "handler_name" {
+  description = "Lambda function handler name"
+  type        = string
+}
+
+variable "apigw_execution_arn" {
+  description = "API Gateway execution arn"
+  type        = string
+  default     = "apigw/GW_EXEC_ARN"
+}
+
+variable "lambda_memory" {
+  description = "Required Memory for Lambda function"
+  default     = 128
+}
+
+variable "security_group_ids" {
+  description = "List of VPC Security Group ids"
+  type        = list(string)
+  default     = []
+}
+
+variable "default_security_group_id" {
+  description = "The ID of the security group created by default on VPC creation"
+  type        = string
+  default     = "vpc/DEFAULT_NSG_ID"
+}
+
+variable "subnets" {
+  description = "List of VPC subnets"
+  type        = string
+  default     = "vpc/LAMBDA_SUBNETS_IDS"
+}
+
+variable "environment_variables" {
+  description = "A map that defines environment variables for the Lambda Function."
+  type        = map(string)
+  default     = {}
+}
+
+variable "ssm_environment_variables" {
+  description = "map of strings containing sensitive SSM parameters; to be passed in as lambda environment variables"
+  type        = map(string)
+}
+
+variable "logs_retention" {
+  description = "Specifies the number of days you want to retain log events in the specified log group"
+  type        = number
+  default     = null
+}
+
+variable "lambda_warmer_rule_name" {
+  description = "Lambda warmer event bridge rule name"
+  type        = string
+}
+
+variable "description" {
+  description = "The description of the rule"
+  type        = string
+}
+
+variable "schedule_expression" {
+  description = "The scheduling expression"
+  type        = string
+}
```

Comment:
Same remark - we should be more explicit in the naming. Maybe lambda_warmer_rule_schedule

Reply:
did